lib/tasks/dev.rake

```diff
@@ -1,3 +1,95 @@
 desc "Fill the database tables with some sample data"
 task({ :sample_data => :environment }) do
+  p "Creating sample data"
+
+  if Rails.env.development?
+    Category.destroy_all
+    Listing.destroy_all
+    Message.destroy_all
+    Offer.destroy_all
+    User.destroy_all
+  end
+
+  12.times do
+    name = Faker::Name.first_name
+    username = Faker::Name.name
+    address = Faker::Address.full_address
+    bio = Faker::Quote.matz
+    location = Faker::Address.city
+    u = User.create(
+      email: "#{name}@example.com",
+      password: "password",
+      username: name,
+      name: username,
+      address: address,
+      image: "https://robohash.org/#{rand(9999)}",
+      bio: bio,
+      location: location,
+    )
+  end
+
+  3.times do
+    category = Category.new
+    categories = Category.create(
+      name: category.name
+    )
+  end
+
+  users = User.all
+
+  users.each do |user|
+    category = Category.new
```

Comment:
I would re-write this line. When you define `category = Category.new`, this is generating 15 new categories rather than adding a random category to your listing. Instead try swapping out line 41 for this: `category = categories.sample # Add random category for each listing`

Reply:
We are defining categories as mentioned in the previous review comment:
![Screenshot 2024-03-26 at 3 45 37 PM](https://github.com/prettyalana/Muse/assets/58926246/7f750380-6e5c-45b8-ba6f-5651d6e330e3)